src/drive/locales/en.json

```diff
@@ -534,6 +535,15 @@
       "noNetwork": "You're currently offline."
     }
   },
+  "Move": {
+    "to": "Move to:",
+    "action": "Move",
+    "cancel": "Cancel",
+    "success": "%{subject} has been moved to %{target}.",
+    "error": "Something went wrong while moving this item, please try again later.",
+    "cancelled": "%{subject} has been moved back to it's original location.",
```

Comment:
Can we display the original path?

Reply:
We could, but it requires another query to get the name of the original folder, since we only have the `id`. That said the original directory should already be loaded in the cozy-client redux store.

I thought it was better to avoid the extra query and just display a message like that. When we start supporting multiple files, we probably won't be able to show the original path either, because there might be more than one.
That said we can change the behavior if you think it's important to have the original path, it's not a big change to make!